pyspectre/core.py

```diff
@@ -75,33 +78,34 @@ def simulate( netlist_path: str, includes: Iterable[str] = None
 
     #ret = os.system(cmd)
     ret = run( cmd
-             , check          = False
-             , stdin          = DEVNULL
-             , stdout         = DEVNULL
-             , stderr         = DEVNULL
-             , capture_output = False
+             , check         =True
+             , stdin         =DEVNULL
+             , stdout        =True
+             , stderr        =True
+             , capture_output=False
```

Comment:
Same issue as above regarding the log. If this is required, we should add `**kwargs` to `simulate` and pass them to `run` so output can be optionally enabled.

Reply:
Agreed!